kong/tools/utils.lua

```diff
@@ -25,7 +25,7 @@ do
     "kong.tools.time",
     -- ]] keep it here for compatibility
     "kong.tools.string",
-    "kong.tools.ip",
+    --"kong.tools.ip",
```

Comment:
revert it before merge, we should keep it for compatibility.

Reply:
yeah, it's reverted.